lib/std/os/bits/wasi.zig

```diff
@@ -4,6 +4,8 @@
 // The MIT license requires this copyright notice to be included in all copies
 // and substantial portions of the software.
 // Convenience types and consts used by std.os module
+pub usingnamespace @import("posix.zig");
+
```

Comment:
I'm pretty sure wasi doesn't have `syslog`, why don't we export only `io_vec` and `iovec_const` as we do for windows?

Reply:
Fair enough!